apps/dashboard/app/onboarding/organization/page.tsx

```
@@ -0,0 +1,280 @@
+  const { user } = usePrivy();
+  const { admin } = useDashboardState();
+  const [companyName, setCompanyName] = useState('');
+  const [organizationHandle, setOrganizationHandle] = useState('');
+  const [referralSource, setReferralSource] = useState('');
+  const [role, setRole] = useState('');
+  const [companyLogo, setCompanyLogo] = useState<string | null>(null);
+  const [logoFile, setLogoFile] = useState<File | null>(null);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleLogoChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.files && event.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)